GooglePlayInstant/Samples/SphereBlast/scripts/BaseGame.cs

```diff
@@ -0,0 +1,272 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using GooglePlayInstant;
+
+public class BaseGame : MonoBehaviour
+{
+	public Transform crate;
+    public Camera thirdPersonCamera;
+    public Camera overheadCamera;
+	private static Vector3 position1 = new Vector3(37, 3.87f, 11);
+    private static Vector3 position2 = new Vector3(37, 3.87f, -4);
+    private static Vector3 position3 = new Vector3(37, 3.87f, -16);
+    private static Vector3 position4 = new Vector3(0, 3.87f, -16);
+    private static Vector3 position5 = new Vector3(0, 3.87f, -4);
+    private static Vector3 position6 = new Vector3(0, 3.87f, 11);
+    private static Vector3 position7 = new Vector3(-23, 3.87f, 11);
+    private static Vector3 position8 = new Vector3(-23, 3.87f, -4);
+    private static Vector3 position9 = new Vector3(-23, 3.87f, -16);
+
+	private int MAX_TIME = 10;
+	private static int MAX_SCORE_PER_LEVEL = 5;
```

Comment:
Make const and change name to suggested name `private const int MaxScorePerLevel = 5;`

Reply:
fixed